FinalProject/FinalProject/src/main/java/com/globant/FinalProject/shopping/ShoppingCart.java

```diff
@@ -0,0 +1,169 @@
+package com.globant.FinalProject.shopping;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+import com.globant.FinalProject.DAO.MySQLConnection;
+import com.globant.FinalProject.DTO.DTOBuy;
+import com.globant.FinalProject.user.User;
+
+/**
+ * This class defines behavior of shopping cart. Allows to add and remove products in the cart, get its total
+ * value and obtain products quantity into the cart. All operations are made in databases.
+ * The products are stored here while the buy is not paid. When the buy is paid, the products are
+ * removed from this table and stored in the TICKET table
+ * @author Herminio Andres Hernandez
+ * @version 04/08/2018
+ */
+public class ShoppingCart {
+	
+	private Connection connection = null;
+	private PreparedStatement preparedStatement = null;
+	private User user;
+	
+	/**
+	 * Constructor of shopping cart.
+	 */
+	public ShoppingCart(User user) throws SQLException {
+		this.user = user;
+		connection = MySQLConnection.getConnection();
+	}
+	/**
+	 * The method executes a query that obtain the quantity of given product.  
+	 * @param idProduct
+	 * @return
+	 * @throws SQLException
+	 */
+	public int getQuantityProduct(String idProduct) throws SQLException {
+		int count = 0;
+		preparedStatement = connection.prepareStatement("SELECT quantity FROM CART WHERE (idProduct=? AND idUser = ?)");
+		preparedStatement.setString(1, idProduct);
+		preparedStatement.setString(2, user.getUser());
+		ResultSet result = preparedStatement.executeQuery();
+		if(result.first()) {
```

Comment:
you are using the cart as not only model but also a service, be careful about that.